.changeset/dry-weeks-compare.md

```diff
@@ -0,0 +1,5 @@
+---
+'@soramitsu-ui/ui': patch
+---
+
+**fix**(`SDatePicker`): add custom trigger support
```

Comment:
"add custom trigger support" sounds like a new feature, not a fix, isn't it?

Reply:
It was in 'documentation' but not implemented. I fixed it